core-bundle/src/InsertTag/Resolver/DateInsertTag.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\InsertTag\Resolver;
+
+use Contao\Config;
+use Contao\CoreBundle\DependencyInjection\Attribute\AsInsertTag;
+use Contao\CoreBundle\InsertTag\InsertTagResult;
+use Contao\CoreBundle\InsertTag\OutputType;
+use Contao\CoreBundle\InsertTag\ResolvedInsertTag;
+use Contao\Date;
+
+#[AsInsertTag('date', asFragment: true)]
+class DateInsertTag implements InsertTagResolverNestedResolvedInterface
+{
+    public function __invoke(ResolvedInsertTag $insertTag): InsertTagResult
+    {
+        $format = $insertTag->getParameters()->get(0) ?? $GLOBALS['objPage']->dateFormat ?? Config::get('dateFormat');
```

Comment:
We have used `$GLOBALS['TL_CONFIG']` instead of `Config::get()` in other new classes, so maybe we want to do the same here?